### DISCUSSION TITLE
Is the _index.html in the exampleSite/content/gallery/ EMPTY for a reason?

### DISCUSSION THREAD

**Discussion by darrenaddy** [Lednerb/bilberry-hugo-theme#369] [Q&A]
It is a 0 Bytes file with zero lines.
(There are no _index.html or index.html files at all in the root of any of the other content types.)

**Comment by igor-baiborodine** [comment]
@darrenaddy, `_index.md` file can be removed. It will work fine without it.

On the other hand, this file can be used to customize the listing of posts of the `gallery` type.

Here, `content/audio/_index.md` and `layouts/_default/list.html` files were customized to display all audio posts as a list similar how it's displayed in the Archive page: https://github.com/igor-baiborodine/bilberry-hugo-theme-sandbox/pull/8/files


Compare these two pages:
1. Your example site: http://localhost:1313/gallery/

2. Bilberry Sandbox: https://deploy-preview-8--bilberry-sandbox.netlify.app/audio/
